### PR TITLE
Adding sku.newPutCommand and unit tests

### PR DESCRIPTION
**Pull request by marosset**
Adding  put command for skus which adds a new sku/plan to an existing offer

### CODE REVIEW

**Comment by marosset** [comment]
Related to #10

**Comment by marosset** [comment]
I want to do some additional testing before this gets merged

**Comment by devigned** [comment]
@marosset let me know when you are ready to move this out of wip or would like a review.

**Comment by marosset** [comment]
I think it is ready for review now - thanks!
I've validated it with some corevm offers but don't know if i have access to any offers with the other type of plan/SKU

File: cmd/sku/put.go [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+		Run: xcobra.RunWithCtx(func(ctx context.Context, cmd *cobra.Command, args []string) error {
+			bits, err := ioutil.ReadFile(oArgs.SkuFilePath)
+			if err != nil {
+				sl.GetPrinter().ErrPrintf("%v\n", err)
+				return err
+			}
+
+			var plan partner.Plan
+			if err := json.Unmarshal(bits, &plan); err != nil {
+				sl.GetPrinter().ErrPrintf("unable to unmarshal JSON plan into partner.Plan: %v", err)
```
Comment (devigned) [inline cmd/sku/put.go]: "Plan" might be weird to a user. Perhaps, this should be SKU since that is the way the CLI talks about this type.

Comment (marosset) [reply cmd/sku/put.go]: Is this better?
"unable to unmarshal JSON from sku file into an object: %v", err
I don't want to mislead by saying we're trying to unmarshall into an object with SKU in the name.

**Pull request merged**